test/cri-containerd/container_test.go

```diff
@@ -215,3 +216,76 @@ func Test_RunContainer_Events_LCOW(t *testing.T) {
 		}
 	}
 }
+
+func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
+	if osversion.Get().Build < 19516 {
+		t.Skip("Requires build +19516")
+	}
+
+	testDeviceInstanceID, err := findTestNvidiaGPUDevice()
+	if err != nil {
+		t.Skipf("skipping test, failed to find assignable nvidia gpu on host with: %v", err)
+	}
+	if testDeviceInstanceID == "" {
+		t.Skipf("skipping test, host has no assignable nvidia gpu devices")
+	}
+
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause, imageLcowAlpine})
+	client := newTestRuntimeClient(t)
+
+	podctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
+	defer cancel()
+
+	sandboxRequest := &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name(),
+				Uid:       "0",
```

Comment:
Is this needed? If so, can you add a comment explaining why?

Reply:
Nope :)